paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/FormUI.kt

```diff
@@ -25,15 +24,14 @@ internal fun FormInternal(
     enabledFlow: Flow<Boolean>,
     elements: List<FormElement>
 ) {
-    val hiddenIdentifiers by hiddenIdentifiersFlow.asLiveData().observeAsState(
+    val hiddenIdentifiers by hiddenIdentifiersFlow.collectAsState(
         null
     )
-    val enabled by enabledFlow.asLiveData().observeAsState(true)
+    val enabled by enabledFlow.collectAsState(true)
 
-    if (hiddenIdentifiers != null) {
+    hiddenIdentifiers?.let {
         Column(
-            modifier = Modifier
-                .fillMaxWidth(1f)
+            modifier = Modifier.fillMaxWidth(1f)
         ) {
             elements.forEach { element ->
                 if (hiddenIdentifiers?.contains(element.identifier) == false) {
```

Comment:
Replace access to `hiddenIdentifiers` with non-optional `it` (or named variable)